jobserver/management/commands/create_project_from_workspace.py

```diff
@@ -27,14 +27,12 @@ def _create_project(self, *, org, workspace):
         # generate a name from the slug
         name = " ".join(word.capitalize() for word in slug.split("-"))
 
-        project = Project.objects.create(org=org, name=name, slug=slug)
+        project = Project.objects.get_or_create(org=org, name=name, slug=slug)
```

Comment:
`get_or_create` returns a tuple, so you need `project, _ = Project...` here
(or `project, new = ` if it's useful to log whether it created a new project or merged with an existing one)

Reply:
🤦‍♂️ Thanks!